tests/e2e/utils/kubeflow_installation.py

```diff
@@ -230,6 +250,28 @@ def install_ack_controller():
     )
 
 
+def configure_kubeflow_pipelines(
+    component_name, installation_paths, installation_option
+):
+    cfg = load_yaml_file(file_path="./utils/pipelines/config.yaml")
+    IAM_ROLE_ARN_FOR_IRSA = cfg["pipeline_oidc_role"]
+    if installation_option == "kustomize":
+        CHART_EXPORT_PATH = "../../apps/pipeline/s3/service-account.yaml"
+        exec_shell(
+            f'yq e \'.metadata.annotations."eks.amazonaws.com/role-arn"="{IAM_ROLE_ARN_FOR_IRSA}"\' '
+            + f"-i {CHART_EXPORT_PATH}"
+        )
+
+    else:
+        IAM_ROLE_ARN_FOR_IRSA = cfg["pipeline_oidc_role"]
+        CHART_EXPORT_PATH = f"{installation_paths}/templates/ServiceAccount/ml-pipeline-kubeflow-ServiceAccount.yaml"
+        exec_shell(
+            f'yq e \'.metadata.annotations."eks.amazonaws.com/role-arn"="{IAM_ROLE_ARN_FOR_IRSA}"\' '
+            + f"-i {CHART_EXPORT_PATH}"
+        )
+        install_helm(component_name, installation_paths)
```

Comment:
(Can do later) Would try to use the install_helm statement in the install_component function if possible.

Reply:
changed to support this